src/dftbp/timedep/linrespgrad.F90

```diff
@@ -133,7 +133,7 @@ subroutine LinRespGrad_old(this, iAtomStart, grndEigVecs, grndEigVal, sccCalc, d
     real(dp), allocatable, intent(inout) :: allOmega(:)
 
     !> difference density matrix (vs. uncharged atoms)
-    real(dp), intent(inout), pointer :: deltaRho(:,:,:)
+    real(dp), intent(inout), optional :: deltaRho(:,:,:)
```

Comment:
This change is the origin of the Furan fail with intel. Either addnadiagradients should be adapted to also make deltaRho optional, or both need to change to it being allocatable.

Reply:
See last commit, but this is not robust. At some point we should check if all optional dummy arguments are actually present before just blindly using them. This concerns multiple places in linrespgrad I guess.

Reply:
I triggered all tests again, see: `Builders/trigger_all_tests/43`